src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -478,9 +481,9 @@ public static void validateOrder(Path path, String name) throws Exception {
             final List<String> names = getNamesFromIndexPage(current);
             final List<String> namesSorted = names.stream()
                     .sorted()
-                    .collect(Collectors.toUnmodifiableList());
+                .collect(Collectors.toUnmodifiableList());
```

Comment:
please restore intentation of chained calls.

Reply:
of course. im sorry.